docs/entity/components.md

```diff
@@ -505,14 +505,18 @@ The Oauth type entity enables us to use Oauth2.0 for user authentication. Visit
 ## `Interval`
 
 A [Text](#text) field used to specify [interval](https://docs.splunk.com/Documentation/Splunk/latest/Admin/Inputsconf#Scripted_Input:)
-value, i.e. a number greater than or equal to 0, or -1.
+value, i.e. a number greater than or equal to 0, CRON interval or -1.
```

Comment:
nit: maybe it's worth mentioning that only basic cron format without month names etc. is supported, as mentioned here:: https://docs.splunk.com/Documentation/Splunk/9.0.2/Admin/Inputsconf#Scripted_Input: